src/test/operators/table_scan_between_test.cpp

```diff
@@ -57,50 +57,91 @@ class TableScanBetweenTest : public TypedOperatorBaseTest {
     _data_table_wrapper = std::make_shared<TableWrapper>(data_table);
     _data_table_wrapper->execute();
   }
+
+  void _test_between_boundaries(std::vector<std::tuple<AllTypeVariant, AllTypeVariant, std::vector<int>>>& tests,
+                                bool left_inclusive = true, bool right_inclusive = true) {
+    const auto& [data_type, encoding, nullable] = GetParam();
+    std::ignore = encoding;
```

Comment:
If we understand correctly, this means the variable is not used.
Using __maybe_unused may be better for this?

Reply:
this code snipped was again just copied. anyway, I have tried:
error: expected ‘]’ before ‘__maybe_unused’
     const auto& [data_type, encoding __maybe_unused, nullable] = GetParam();

https://stackoverflow.com/questions/40673080/stdignore-with-structured-bindings
It seems like I can either mark all of the variables as maybe_unused, declare them separately or just keep the current solution.